test/python/transpiler/test_permrowcol.py

```diff
@@ -104,6 +112,20 @@ def test_perm_row_col_doesnt_return_cnots_with_identity_matrix_permutation(self)
         instance = permrowcol.perm_row_col(parity_mat)[0]
         self.assertEqual(len(instance.data), 0)
 
+    def test_perm_row_col_returns_valid_output_with_a_common_case(self):
```

Comment:
This and some of the imports seem to be added also in #23?

Reply:
It is because we merged the other branch to the other, for we were using it as a base to create the other test. I don't think there should come any conflict with merge, but we can edit those, if it is necessary.